src/app/cli/src/init/client.ml

```diff
@@ -56,19 +56,26 @@ let get_balance =
   let address_flag =
     flag "address"
       ~doc:"PUBLICKEY Public-key for which you want to check the balance"
-      (required Cli_lib.Arg_type.public_key)
+      (required Cli_lib.Arg_type.public_key_compressed)
+  in
+  let token_flag =
```

Comment:
if we're hiding this from GraphQL should we also be hiding it in CLI? Which release are you targetting this one for?